Mirror/Runtime/Transport/EpicOnlineTransport/EOSSDK/Core/Config.cs

```diff
@@ -59,8 +59,8 @@ public static class Config
 		#elif EOS_PLATFORM_WINDOWS_64
 			"EOSSDK-Win64-Shipping.dll"
 
-		#elif EOS_PLATFORM_OSX && EOS_UNITY
-			"libEOSSDK-Mac-Shipping"
+		#elif UNITY_EDITOR_OSX
+			"libEOSSDK-Mac-Shipping.dylib"
```

Comment:
Since the string is the same in the editor and in builds, we don't need separate checks any more. Just one #elif EOS_PLATFORM_OSX should suffice